### ISSUE TITLE
Add dropdown filter so that users can filter by time period (years)

### ISSUE THREAD

**Issue by brittanyrw** [brittanyrw/emojiscreen#141]
In the top of EmojiScreen in the filters section, add a dropdown filter so that users can filter by years. The filter options should be by decades such as 1970s, 1980s, 1990s, 2010s, etc.

Match the styling to the current filters and use the existing colors on the website. You may need to reconfigure the HTML for the filters section to add in this new dropdown.

If there are no items that match the time period selected, a message should appear saying that no cards match the selected years.

One solution for reorganizing is to add an 'advanced filters' section that is collapsable so that you can add the dropdown in an advanced filters section under the current filters and hide it unless clicked on. Eventually we will also want to add a filter for genres as well so this would be a good solution to ensure it would be easy to expand and add more filters without cluttering the homepage.

**Comment by cjtangney** [comment]
I'd be happy to take care of this. I should have some time this evening or over the weekend to crank this out 😃

**Comment by cjtangney** [comment]
I noticed another issue requesting someone to pull jQuery out from `app.js`

Would you prefer the filtering functionality be written in vanilla JS, or would you like me to lean on jQuery for the time-being?

To me, it certainly makes more sense to be pushing vanilla JS on you if you're looking to rip jQuery out of the application in the future! 😄

**Comment by brittanyrw** [comment]
@cjtangney Thanks for taking this.

Ah yes, there are a couple of features in progress that I think are using jQuery so I haven't merged in the update someone already did to switch everything to vanilla JS yet. 

Feel free to write in vanilla JS if you prefer. If it is easier, you can write it in jQuery for now and we can translate it over later.

**Comment by cjtangney** [comment]
Would you mind if I used template literals in favor of string concatenation?

**Comment by brittanyrw** [comment]
Sure, that works!

**Comment by cjtangney** [comment]
This looking okay from a visual perspective?

![image](https://user-images.githubusercontent.com/38563318/64576558-20539c80-d347-11e9-8670-d2a02a1bc4a9.png)

![image](https://user-images.githubusercontent.com/38563318/64576566-2e092200-d347-11e9-9d3d-cc1253281a2c.png)

I tried to stick with the theme of the application as closely as possible. The colors for the actual buttons themselves are not the best, but I had to go with something!

The filter is a mess of jQuery / vanilla right now. I will probably make a first pass at the genre filter as well before converting it all to vanilla.

If you have any colors you'd like me to use for the buttons specifically, let me know!
